rpc/client/evidence_test.go

```diff
@@ -131,6 +125,9 @@ func TestBroadcastEvidence_DuplicateVoteEvidence(t *testing.T) {
 		correct, fakes := makeEvidences(t, pv, chainID)
 		t.Logf("client %d", i)
 
+		// make sure that the node has produced enough blocks
+		waitForBlock(t, ctx, c, 2)
```

Comment:
Nice! Not right now, but I think we can do this in most places in tests like this where we use sleeps.